code-challenges.test.js

```diff
@@ -11,39 +11,108 @@
 // Run the file with the following command: $ yarn jest
 
 
+
 // --------------------1) Create a function that takes in an array, removes the first item from the array and shuffles the remaining content.
 
 // a) Create a test with an expect statement using the variable provided. HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
-const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+describe("colorHouse", () => {
+
+    const colors1 = ["purple", "blue", "green", "yellow", "pink"]
 // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
 const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
+    it("Create a function name colorHouse, colorHouse will return different array length of colors, the first color in the array will be removed, the rest will get placed in random order", () => {
+
+        expect(colorHouse(colors1)).toEqual(("can be a different order") ["yellow", "blue", "pink", "green"])
+  })
+})
+
 
 // b) Create the function that makes the test pass.
 
+// Create a function colorHouse
+// colors1 and colors2, first index will be dropped
+// The rest of colors1 and colors2 will return in an array in random order    
```

Comment:
Would really like to see you expand your pseudocode next time around.  Pseudocode should provide a detailed approach and road map to how you will write your function.